layers/+lang/latex/packages.el

```diff
@@ -77,10 +77,16 @@
         "mv" 'TeX-view))))
 
 (when (string= latex-build-command "LatexMk")
-(defun latex/init-auctex-latexmk ()
-  (use-package auctex-latexmk
-    :defer t
-    :init (add-hook 'LaTeX-mode-hook 'auctex-latexmk-setup))))
+  (defun latex/init-auctex-latexmk ()
+    (use-package auctex-latexmk
+      :defer t
+      :init
+      (progn
+        (add-hook 'LaTeX-mode-hook
+                  (lambda()
+                    (setq TeX-command-default "LatexMk")))
+        (auctex-latexmk-setup)
```

Comment:
I'm not sure what hook you are referring to, or what is outside of it. I was mostly trying to keep the structure that already existed, just adding what needed to be added to make `LatexMk` work by default.